Don't automatically activate the Apache PHP object by editing
Apache's httpd.conf.
We choose to let the user do that manually.

Will not send this upstream.


diff --git a/sapi/apache2handler/config.m4 b/sapi/apache2handler/config.m4
index bcdb188..ddb7ea3 100644
--- a/sapi/apache2handler/config.m4
+++ b/sapi/apache2handler/config.m4
@@ -64,7 +64,7 @@ if test "$PHP_APXS2" != "no"; then
   fi
 
   APXS_LIBEXECDIR='$(INSTALL_ROOT)'`$APXS -q LIBEXECDIR`
-  if test -z `$APXS -q SYSCONFDIR`; then
+  if test -z ''; then
     INSTALL_IT="\$(mkinstalldirs) '$APXS_LIBEXECDIR' && \
                  $APXS -S LIBEXECDIR='$APXS_LIBEXECDIR' \
                        -i -n php"
